.github/workflows/djangoplicity.yml

```
@@ -29,7 +29,7 @@ jobs:
         pip install -r requirements.txt
     - name: Run Tests
       run: |
-        coverage run --source='.' manage.py test
+        coverage run --source='.' manage_test.py test
```

Again, I believe just setting the Env variable should work:
```suggestion
        env DJANGO_SETTINGS_MODULE="test_project.dp_test_settings" coverage run --source='.' manage_test.py test
```